tox.ini

```diff
@@ -1,7 +1,8 @@
 [tox]
 envlist =
-    {py37,py38,py39}-django{22,31}
+    {py37,py38,py39}-django22
     {py37,py38,py39,py10}-django32
+    {py38,py39,py10}-djangomain
```

Comment:
I think it should be better to explicit the version (`django40`) to know what it is going on without checking another repo.

Reply:
I added Django4.0 to the test pipeline.